test_results_win64.csv

```diff
@@ -60,7 +60,7 @@ Standalone Battery,Merchant Plant,-8397703,22.1921,NA,;
 Standalone Battery,Leveraged Partnership Flip,-8397703,82.8137,88.6283,; 
 Standalone Battery,All Equity Partnership Flip,-8397703,86.5441,88.6283,; 
 Standalone Battery,Sale Leaseback,-8397703,86.3995,88.6283,; 
-ETES,Single Owner,-498870720,17.93,12.4508,; 
+ETES,Single Owner,-497823968,17.9392,12.4517,; 
```

Comment:
Any idea why these are changing? This is a SAM meeting agenda item

Reply:
Maybe recent changes in SSC without updates to test results? For example, https://github.com/NREL/ssc/commit/5110e354141687560811be5c62a877d3fe5634b0.